plugins/techdocs/package.json

```diff
@@ -44,11 +44,13 @@
     "@backstage/integration-react": "^1.0.0",
     "@backstage/plugin-catalog-react": "^1.0.0",
     "@backstage/plugin-search": "^0.7.4",
+    "@backstage/plugin-techdocs-module-mkdocs": "^0.0.1",
```

Comment:
I wonder if it make sense to only depend on the techdocs-module-mkdocs plugin from the App, where the app integrator compose their App?